lib/galaxy/tools/data/__init__.py

```diff
@@ -675,13 +675,13 @@ def filter_file_fields(self, loc_file, values):
             rval = ""
             for line in reader:
                 if line.lstrip().startswith(self.comment_char):
-                    rval += line
+                    rval += line.encode('utf-8')
```

Comment:
Can you write the file in text mode instead ? `open(loc_file, 'w')` on line 686 ?